devenv.yaml

```diff
@@ -2,7 +2,7 @@ inputs:
   nixpkgs:
     url: github:NixOS/nixpkgs/nixpkgs-unstable
   nix:
-    url: github:domenkozar/nix/relaxed-flakes
+    url: github:domenkozar/nix/relaxed-flakes-2.17
```

Comment:
What is relaxed-flakes?

Reply:
There are a few patches we need for Nix (that will one day land upstream) to make devenv work.

I've ported them to Nix 2.17, but there's still a bug.